pkgs/development/python-modules/lox/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+
+  pathos,
+  pip,
+  pytestCheckHook,
+  pytest-benchmark,
+  pytest-mock,
+  pytest-runner,
+  tqdm,
+}:
+
+buildPythonPackage rec {
+  pname = "lox";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "BrianPugh";
+    repo = "lox";
+    rev = "v${version}";
+    hash = "sha256-Iv3ZdfsvFLU6lhlH1n+eQ+KIrXESsnC1S2lVFnKFV08=";
+  };
+
+  propagatedBuildInputs = [ pathos ];
+
+  buildInputs = [ pip ];
```

Comment:
Why does it ave `pip` as `buildInputs`?

Reply:
When I was doing the initial development, it was explicitly calling `pip` during the build. It looks like that might have been something that was improperly passing through the pytest runs. Removing this now still results in the library building fine.